### DISCUSSION TITLE
windows defender suddenly mark dxgi.dll as Trojan:Win32/Spursint.Q!cl

### DISCUSSION THREAD

**Discussion by kemudian** [valinet/ExplorerPatcher#264] [Q&A] [answered]
explorer crash and after i tried to run explorer.exe from task manager it give me 0xc0000906 error 

EDIT: just restarted, dll still there and the patch still work even though i didn't touch anything in windows security, protection history still show the file in quarantined no action has been taken

**Comment by DustyMustard** [comment]
I get this as well. Installed yesterday and Properties under windows key + x worked.

Today I try that again and this happens. Windows just tells me it found a suspicious file, explorer closes, and if I try to relaunch explorer it says the 0xc0000906 error. Only way to get explorer back is if I wait some time before restarting, as if I restart right after the first virus warning message it will boot into Windows without explorer and launching it via task manager won't work.

**Comment by valinet** [comment, marked as answer]
I understand. Windows Defender is not a product that I have any control over. The user has some control over it, namely adding an exception for a particular file. I think that may work well in this case.

Related: https://github.com/valinet/ExplorerPatcher/issues/265, including a tutorial on how to ad an exception to Windows Defender.

Thanks

**Reply by gailium119** [reply]
I think some functions of the project are marked unsafe by WD.When I in-build the 22000.194 EP into the project, WD says it as a trojian ,which later I solved it by discovering an error that I  swapped two EPs' value by mistake .
